doc/InfiniTimeVision.md

```diff
@@ -0,0 +1,32 @@
+# InfiniTime Vision
+
+The purpose of this document is to steer efforts towards a common goal, and as such should be taken into consideration with all developments.
+
+## InfiniTime
+
+InfiniTime is a community built smartwatch firmware, providing freedom and privacy that's usually insufficient in wearable technology.
+
+InfiniTime is not to be used for medical or other health tracking purposes.
+
+## Core Principles
+
+- KISS (Keep It Short and Simple)
+- Reliability
+- Battery efficiency
+- Easy and simple navigation
+- Behaviour should be predictable and easy to understand
+- Prefer solid default experience over customization
+- Personalization is achieved through custom watchfaces.
+More options may be available through a companion app.
+- Use standard protocols and methods
+
+## Long term vision
+
+The perfect version of InfiniTime would include:
+
+- Capability to upload apps and watchfaces from a phone
+- Only minimal features.
+Users would add the features they want.
```

Comment:
It seems to render fine without spaces, but in the text it is easer to read if indented
```suggestion
  Users would add the features they want.
```